libs/SoloPlay/BuildFiles/sorceress.BlovaBuild.js

```diff
@@ -21,15 +21,7 @@ const finalBuild = {
 	skills: [
 		[sdk.skills.Warmth, 1],
 		[sdk.skills.FrozenArmor, 1],
-		[sdk.skills.IceBolt, 1],
-		[sdk.skills.IceBlast, 1],
-		[sdk.skills.StaticField, 1],
-		[sdk.skills.Telekinesis, 1],
-		[sdk.skills.FrostNova, 1],
 		[sdk.skills.Teleport, 1],
-		[sdk.skills.GlacialSpike, 1],
-		[sdk.skills.Blizzard, 1],
-		[sdk.skills.ColdMastery, 1],
 		[sdk.skills.Nova, 20],
```

Comment:
Should stick at least 1 point into Blizz to ensure having it before moving to max out other skills

Reply:
my bad, I did a bad copy/paste